### PR TITLE
trafficgraph, dont update the on screen visual graph while invisible

### PR DESCRIPTION
**Pull request by PiBa-NL**
trafficgraph, dont update the on screen visual graph while invisible (which avoids creating a large queue of pending timer objects waiting for the next requestAnimationFrame to happen)

Should fix: https://redmine.pfsense.org/issues/7157

### CODE REVIEW

File: src/usr/local/www/vendor/nvd3/nv.d3.js [hunk 1]
```diff
@@ -1949,7 +1949,10 @@ nv.utils.arrayEquals = function (array1, array2) {
             scale = _;
             axis.scale(scale);
             isOrdinal = typeof scale.rangeBands === 'function';
-            nv.utils.inheritOptionsD3(chart, scale, ['domain', 'range', 'rangeBand', 'rangeBands']);
+
+            //code 'leaks' memory. (builds an ever groing string of the same options..)
```
Comment (phil-davis) [inline src/usr/local/www/vendor/nvd3/nv.d3.js]: groing? :)
growing

Comment (PiBa-NL) [inline src/usr/local/www/vendor/nvd3/nv.d3.js]: Thanks :)

**Comment by doktornotor** [comment]
@PiBa-NL  Thanks, that definitely helped here.

**Comment by rbgarga** [comment]
Can you please take a look at it @jdillard?

**Comment by rbgarga** [comment]
@PiBa-NL I've reported nvd3 issue upstream and there is a proposed patch to fix it here https://github.com/novus/nvd3/pull/1957/files

**Comment by rbgarga** [comment]
Partially committed on 67efd86256771a4daa08ed5476f1565b4fb1e5ed after cherry-pick upstream fix on ad1e294ce5a735b680aa9ef93f25906077c8812d

Thanks!

**Comment by jdillard** [comment]
@PiBa-NL Thanks for taking the time to dig into the root of the issue and making nvd3 and pfSense better. Great work!

**Pull request closed without merging**